db/migrate/20171027124746_add_timestamps_and_remove_date_to_poll_recount.rb

```diff
@@ -0,0 +1,8 @@
+class AddTimestampsAndRemoveDateToPollRecount < ActiveRecord::Migration
+  def change
+    change_table :poll_recounts do |t|
+      t.remove :date
+      t.date :created_at
```

Comment:
It would be better to use a time column to store the time as well as date, since many times the admins want to debug some problem and having the time of creation can be helpful (as well at the PartialResult object).